packages/field/src/react/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default } from './field'
 export * from './field'
+export { FieldInputProps } from './input'
```

Comment:
I actually think we need to export this for all **Field** subcomponents the **typeof pattern** get's weird when it's working off a component that's wrapped in ForwardRef and we will need this interface building form inputs  off field.

Reply:
What I think you're saying is it is not used today but that we're going to use it later as input components are refactored to use Field. Is that the case? 

If so, I think that's fine, but it seems like it doesn't belong in this PR because it's not cohesive with the purpose of the PR. Do you agree?

Reply:
Sure I'll take it out and create a card linked to this ticket. I think I was using it but realized I could type better and left it there.